src/main/java/de/zalando/aruha/nakadi/validation/FieldNameMustBeSet.java

```diff
@@ -5,16 +5,21 @@
 import de.zalando.aruha.nakadi.domain.EventType;
 import de.zalando.aruha.nakadi.domain.ValidationStrategyConfiguration;
 
+import java.util.Optional;
+
 public class FieldNameMustBeSet extends ValidationStrategy {
 
     public static final String NAME = "field-name-must-be-set";
     private final EventValidator executor = new EventValidator() {
 
         @Override
-        public boolean accepts(final JSONObject event) {
-            return event.get("name") != null;
+        public Optional<ValidationError> accepts(final JSONObject event) {
+            if (event.has("name") && event.get("name") != null) {
```

Comment:
I think so. The first check `event.has("name")` checks if there is such a key in the object. But the value could still be `null`.